include/oneapi/dpl/pstl/utils.h

```diff
@@ -48,6 +48,30 @@ namespace dpl
 namespace __internal
 {
 
+#if _ONEDPL_BACKEND_SYCL
+
+template <typename... _Ts>
```

Comment:
In the full PR, we need comments for how this is to be used:
"__is_types_device_copyable is used to define which template parameters are required to be sycl device copyable to make the type being specialized device copyable.  Generally, this means the template arguments which are used as member variables of the type being specialized."